README.md
# lbry-react
React LBRY app


history.txt
    1  sudo apt-get install chromium-browser 
    2  x-special/nautilus-clipboard
    3  copy
    4  file:///home/jack/Downloads/google-chrome-stable_current_amd64.deb
    5  'x-special/nautilus-clipboard
    6  copy
    7  file:///home/jack/Downloads/google-chrome-stable_current_amd64.deb
    8  '
    9  google-chrome-stable_current_amd64.deb
   10  suydo apt install ./google-chrome-stable_current_amd64.deb
   11  sudo apt install ./google-chrome-stable_current_amd64.deb
   12  sudo apt-get install xubuntu-desktop
   13  ln -s /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/Downloads /home/jack/Downloads
   14  cd ~/
   15  echo "ln -s /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/Downloads /home/jack/Downloads" >>symlinks.info
   16  ln -s /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/briefcase /home/jack/Desktop/briefcase
   17  ln -s /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/Storage /home/jack/Storage
   18  ln -s /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/Storage /home/jack/Desktop/Storage
   19  free
   20  cat /proc/cpuinfo
   21  htop
   22  sudo snap install htop
   23  htop
   24  sudo apt install sysbench
   25  sysbench
   26  sudo apt-get update
   27  sudo apt-get upgrade
   28  sudo reboot now
   29  sudo smartctl /dev/sda -a
   30  apt search smartctl
   31  sudo hdparm -B254 /dev/sda
   32  sudo install gsmartcontrol
   33  sudo apt install gsmartcontrol
   34  sudo fstrim -av
   35  free
   36  top
   37  apt search libpng12-0
   38  sudo +x install.sh
   39  chmod +x install.sh
   40  ./install.sh
   41  vi
   42  chmod +x SynfigStudio
   43  cp SynfigStudio /usr/locaal/bin/
   44  cp SynfigStudio /usr/locaal/bin/SynfigStudio
   45  cp SynfigStudio /usr/local/bin/SynfigStudio
   46  sudo cp SynfigStudio /usr/local/bin/SynfigStudio
   47  visudo
   48  sudo visudo
   49  setxkbmap -option
   50  SynfigStudio
   51  apt search synfig
   52  sudo apt install synfig
   53  synfig
   54  sudo mv share sharebak
   55  ls share
   56  sudo cp sharebak share
   57  sudo cp -r sharebak share
   58  ls share
   59  sudo reboot now
   60  sudo ln -s /usr/share /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/share
   61  pwd
   62  ls -n /home/jack/Desktop
   63  ls -n /home/jack/Desktop/Briefcase /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/briefcase
   64  ls -n  /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/briefcase /home/jack/Desktop/Briefcase 
   65  ls -n  /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/briefcase /home/jack/Desktop/briefcase 
   66  sudo update db
   67  sudo updatedb
   68  locate briefcase
   69  ln -s /home/jack/Desktop/Briefcase /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/briefcase
   70  ls
   71  ln -s  /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/briefcase /home/jack/Desktop/briefcase 
   72  ln -s  /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/Downloads /home/jack/Downloads 
   73  ln -s  /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/share /home/jack/Desktop/Share 
   74  ln -s  /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/Storage /home/jack/Desktop/Storage 
   75  ln -s  /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/Downloads /home/jack/Downloads 
   76  nano screencap
   77  ./screencap
   78  chmod +x screencap
   79  screencap
   80  ./screencap
   81  sudo apt install flowblade
   82  ffmpeg
   83  sudo apt  install ffmpeg
   84  ffmpeg
   85  ./screencap
   86  sub
   87  $display
   88  echo $Display
   89  sub
   90  sudo updatedb
   91  locate /sub
   92  locate bin/sub
   93  sub
   94  locate bin/sub
   95  /usr/local/bin/sub
   96  chmod +x /usr/local/bin/sub
   97  rm /usr/local/bin/sub
   98  rm /usr/local/bin/sublime_text
   99  ls /usr/local/bin/sublime*
  100  rm /usr/local/bin/sublime_textext
  101  ln -s /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/Applications/sublime_text_3/sublime_text /usr/local/bin/sub
  102  sub
  103  chmod +x sublime_text
  104  ln -s /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/Applications/sublime_text_3/sublime_textext /usr/local/bin
  105  sudo chown -r jack /usr/local/bin 
  106  sudo chown jack /usr/local/bin 
  107  ln -s /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/Applications/sublime_text_3/sublime_textext /usr/local/bin
  108  sublime_text
  109  ln -s /mnt/c17e104c-58c7-47bb-8cf2-3752068b9c33/Applications/sublime_text_3/sublime_textext /usr/local/bin/sub
  110  sub
  111  ls -rant /usr/local/bin
  112  screencap
  113  ./screencap
  114  sub screencap
  115  ./screencap
  116  ./screencap 001
  117  sudo add-apt-repository ppa:videolan/master-daily
  118  sudo apt install vlc
  119  apt search brave
  120  nano
  121  chown -r jack *.*
  122  chown jack *.*
  123  chown jack Art
  124  chown -R jack Art
  125  sudo chown -R jack Art
  126  sudo apt install apt-transport-https ca-certificates curl software-properties-common
  127  curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
  128  sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu focal stable"
  129  sudo apt update
  130  apt-cache policy docker-ce
  131  sudo apt upgrade
  132  sudo apt install docker-ce
  133  sudo systemctl status docker
  134  sudo usermod -aG docker jack
  135  docker -ps
  136  mkdir Art
  137  cd Art
  138  docker run -v $(pwd)/samples:/nd/samples -v $(pwd)/frames:/nd/frames                          -it alexjc/neural-doodle
  139  sudo docker run -v $(pwd)/samples:/nd/samples -v $(pwd)/frames:/nd/frames                          -it alexjc/neural-doodle
  140  ls
  141  echo "sudo docker run -v $(pwd)/samples:/nd/samples -v $(pwd)/frames:/nd/frames -it alexjc/neural-doodle" >>makeart
  142  docker pull dorowu/ubuntu-desktop-lxde-vnc:focal
  143  sudo docker pull dorowu/ubuntu-desktop-lxde-vnc:focal
  144  sudo docker run --name ubvnc -p 6080:80 -p 5900:5900 dorowu/ubuntu-desktop-lxde-vnc:bionic
  145  sudo apt-get install tigervnc-viewer
  146  sub ~/.bashrc
  147  docker -ps
  148  docker -images
  149  sudo docker -images
  150  sudo docker -ps
  151  sudo docker ps
  152  docker stop 6797ecf5fa3d

src/App-bad.js
import logo from './LBRY.png';
import $ from "jquery";
import './App.css';


window.onload = function() {
  var resetStorage = document.getElementById("emptyStorage");
  resetStorage.onclick = clearStorage;
  }
  function clearStorage() {
  localStorage.clear();
  alert("Storage is Empty");
  }
  // POP UP INSTRUCTIONS
  
  alert("The small fish floats randomly. Use  ' s ',  ' e ',  ' f ' & ' d '  keys to rapidly move the large fish. Arrows keys move one pixel at a time. Fish 'Kiss' with a heart and score, when placed lip to lip. A 'Kiss' has one pixel tolerance it is NOT easy.");
        /**
         * Delay for a number of milliseconds
         */
        function sleep(delay) {
          var start = new Date().getTime();
          while (new Date().getTime() < start + delay);
        }
  var score = localStorage.getItem("highscore", score);
  //document.getElementById("currentnote").innerHTML = score;
  //console.log("pre function: " + score);
  function functionADD() {
  //console.log("after function: " + score);
  document.getElementById("currentnote").innerHTML = score;
  score++;
  sleep(500);
  console.log("After Increment: " + score);
  //   localStorage.setItem("score", JSON.stringify(score));
  var highscore = localStorage.getItem("highscore", score);
  if(highscore !== null){
     if (score > highscore) {
        localStorage.setItem("highscore", score );
        }
  }else{
        localStorage.setItem("highscore", score );
  }
  }
  
  var mxt = 0;
  var gx = 0;
  do {
        function GetBox () {
          var div = document.getElementById ("floater");
              if (div.getBoundingClientRect) {        // Internet Explorer, Firefox 3+, Google Chrome, Opera 9.5+, Safari 4+
                var rect = div.getBoundingClientRect ();
                mx = rect.left;
                my = rect.top;
                mw = rect.right - rect.left;
                mh = rect.bottom - rect.top;
                //The collision calibrates different for each collision / kiss. You Must slow down both fish to 'calibrate'
               mmt = my+78;
               mxt= mx + 135;
  
               
               // floater mmt = my+78;
               // floater mxt= mx + 135;
              //  myraright mmt = my+58;
              //  myraright mxt= mx + 194;
              //  document.getElementById("left").innerHTML = (" Left: " + Math.round(mxt) + "\n Top: " + Math.round(mmt) + "\n Width: " + mw + "\n Height: " + mh);
  //               console.log(Math.round(mxt) );
  
                   document.getElementById("left").innerHTML = (" Left: " + Math.round(mxt) + "\n Top: " + Math.round(mmt) );
              }
              else {
                alert ("Your browser does not support this example!");
              }             //document.querySelector('#myraright').innerHTML = mmt;
        }
  // second segment of do loop ---  start get ElementById image1
            function GetFish () {
             var div = document.getElementById ("image1");
              if (div.getBoundingClientRect) {        // Internet Explorer, Firefox 3+, Google Chrome, Opera 9.5+, Safari 4+
                var rect = div.getBoundingClientRect ();
                gx = rect.left;
                gy = rect.top;
                w = rect.right - rect.left;
                h = rect.bottom - rect.top;
                var  mt = gy+165;
                rightLip =  gx + (mt + 155) 
            //    document.getElementById("right").innerHTML = (" Right: " + gx + "\n Top: " + mt + "\n Width: " + w + "\n Height: " + h);
               document.getElementById("right").innerHTML = (" Right: " + gx + "\n Top: " + mt );
               
                if (Math.round(mxt)+1 == gx && Math.round(mmt) == mt) {
                  functionADD();
                  tempAlert("<h1> KISS ! </h1>",2000);
                  function tempAlert(msg,duration) 
                  {
                   var el = document.createElement("div");
                   el.setAttribute("style","position:absolute;top:40%;left:25%;background-image:url('heart.png');background-repeat: no-repeat;");
                   el.innerHTML = msg;
                   setTimeout(function(){
                    el.parentNode.removeChild(el);
                   },duration);
                   document.body.appendChild(el);
                  }
                }
              }
              else {
                alert ("Your browser does not support this program");
              }
            }
         // End get ElementById image1 second segment of do loop ---  
  
         // set interval between getting position of selections 
              setInterval(GetFish, 5);
              setInterval(GetBox, 5);
              }
              while(mxt != gx && mmt != mt){
   }
   /// End of do loop
  
      $(document).ready(function(){
          animateDiv();
      });
  function float(){
      // Using lesson: Getting Window Dimensions
        var h = $(window).height() - 75;
        var w = $(window).width() -35; 
    // Use var h and w defined for a specific area of float
      //  var h = 100;
      //  var w = 200;
       //The +200 and +500 is to set them apart yet facing each other without too much overlap 
        var nh = Math.floor(Math.random() * h);
        var nw = Math.floor(Math.random() * w);
        return [nh,nw];    
  }
  
  function animateDiv(){
      var newq = float();
      var oldq = $('.a').offset();
      var speed = calcSpeed([oldq.top, oldq.left], newq);
      $('.a').animate({ top: newq[0], left: newq[1] }, speed, function(){
        animateDiv();        
      });
  };
  function calcSpeed(prev, next) {
      var x = Math.abs(prev[1] - next[1]);
      var y = Math.abs(prev[0] - next[0]);
      var greatest = x > y ? x : y;
      var speedModifier = .03;
      var speed = Math.ceil(greatest/speedModifier);
      return speed;
  }
  
  
      $(document).ready(function(){
      animateDivm();
   });
  function float1(){
      // if you wish  Window - viewport dimensions
   var hm = $(window).height() - 75;
   var wm = $(window).width() - 25;  
   //   var hm = 100;
   //   var wm = 200;
      var nhm = Math.floor(Math.random() * hm -150);
      var nwm = Math.floor(Math.random() * wm - 200);
      return [nhm,nwm];    
   }
  
  function animateDivm(){
      var myral = float1();
      var oldq = $('.b').offset();
      var speed = calcSpeed2([oldq.top, oldq.left], myral);
          $('.b').animate({ top: myral[0], left: myral[1] }, speed, function(){
        animateDivm();        
      });
  };
  
  function calcSpeed2(prev, next) {
      var x = Math.abs(prev[1] - next[1]);
      var y = Math.abs(prev[0] - next[0]);
      var greatest = x > y ? x : y;
      var speedModifier = .01;
      var speed = Math.ceil(greatest/speedModifier);
      return speed;
  }
  //Start Animate .c
   $(document).ready(function(){
      animateDivC();
   });
  function floatC(){
      // if you wish  Window - viewport dimensions
   var hm = $(window).height() - 75;
   var wm = $(window).width() - 75;  
   //   var hm = 100;
   //   var wm = 200;
      var nhm = Math.floor(Math.random() * hm -50);
      var nwm = Math.floor(Math.random() * wm - 100);
      return [nhm,nwm];    
   }
  
  function animateDivC(){
      var newC = floatC();
      var oldC = $('.c').offset();
      var speed = calcSpeedC([oldC.top, oldC.left], newC);
          $('.c').animate({ top: newC[0], left: newC[1] }, speed, function(){
        animateDivC();        
      });
  };
  
  function calcSpeedC(prev, next) {
      var x = Math.abs(prev[1] - next[1]);
      var y = Math.abs(prev[0] - next[0]);
      var greatest = x > y ? x : y;
      var speedModifier = .03;
      var speed = Math.ceil(greatest/speedModifier);
      return speed;
  }
  //End Animate .c
  //Start .d
   $(document).ready(function(){
      animateDivD();
   });
  function floatD(){
      // if you wish  Window - viewport dimensions
   var hm = $(window).height();
   var wm = $(window).width() ;  
   //   var hm = 100;
   //   var wm = 200;
      var nhm = Math.floor(Math.random() * hm );
      var nwm = Math.floor(Math.random() * wm );
      return [nhm,nwm];    
   }
  
  function animateDivD(){
      var newC = floatD();
      var oldC = $('.d').offset();
      var speed = calcSpeedD([oldC.top, oldC.left], newC);
          $('.d').animate({ top: newC[0], left: newC[1] }, speed, function(){
        animateDivD();        
      });
  };
  
  function calcSpeedD(prev, next) {
      var x = Math.abs(prev[1] - next[1]);
      var y = Math.abs(prev[0] - next[0]);
      var greatest = x > y ? x : y;
      var speedModifier = .04;
      var speed = Math.ceil(greatest/speedModifier);
      return speed;
  }
  //End Animate .d
  
  $(function(){
          var x = 0;
          setInterval(function(){
              x-=1;
              $('body').css('background-position', x + 'px 0');
          }, 300);
      })
  
  function myFunction() {
      location.reload();
  }
  
  
  window.addEventListener("storage", function(e) {
     console.debug(e);
  }, false);
  
  

  function saveStuff() {
      sessionStorage.setItem('sessionKey', 'sessionValue');
      localStorage.setItem('localKey', 'localValue');
  }
  localStorage.getItem('localKey');
  sessionStorage.getItem('sessionKey');
  
  
  
  var score = localStorage.getItem("score");

  
    var ie=document.all;
    var nn6=document.getElementById&&!document.all;
    var isdrag=false;
    var x,y;
    var dobj;
    function movemouse(e)
    {
      if (isdrag)
      {
        dobj.style.left = nn6 ? tx + e.clientX - x : tx + event.clientX - x;
        dobj.style.top  = nn6 ? ty + e.clientY - y : ty + event.clientY - y;
        return false;
      }
    }
  
    function selectmouse(e) 
    {
      var fobj       = nn6 ? e.target : event.srcElement;
      var topelement = nn6 ? "HTML" : "BODY";
  
      while (fobj.tagName != topelement && fobj.className != "dragme")
      {
        fobj = nn6 ? fobj.parentNode : fobj.parentElement;
      }
  
      if (fobj.className=="dragme")
      {
        isdrag = true;
        dobj = fobj;
        tx = parseInt(dobj.style.left+0);
        ty = parseInt(dobj.style.top+0);
        x = nn6 ? e.clientX : event.clientX;
        y = nn6 ? e.clientY : event.clientY;
        document.onmousemove=movemouse;
        return false;
      }
    }
    document.onmousedown=selectmouse;
    document.onmouseup=new Function("isdrag=false");
  

      $( ".fastclick" ).bind( "tap", function( e ){
        alert( "Tap!" );
      });



var mxt = 0;
var gx = 0;
var mt = 0;
var mnt = 0;
var mmt = 0;
do {
      function GetBox () {
        var div = document.getElementById ("floater");
            if (div.getBoundingClientRect) {        // Internet Explorer, Firefox 3+, Google Chrome, Opera 9.5+, Safari 4+
              var rect = div.getBoundingClientRect ();
              mx = rect.left;
              my = rect.top;
              mw = rect.right - rect.left;
              mh = rect.bottom - rect.top;
              //The collision calibrates different for each collision / kiss. You Must slow down both fish to 'calibrate'
             mmt = my+78;
             mxt= mx + 135;

             
             // floater mmt = my+78;
             // floater mxt= mx + 135;
            //  myraright mmt = my+58;
            //  myraright mxt= mx + 194;
            //  document.getElementById("left").innerHTML = (" Left: " + Math.round(mxt) + "\n Top: " + Math.round(mmt) + "\n Width: " + mw + "\n Height: " + mh);
//               console.log(Math.round(mxt) );

                 document.getElementById("left").innerHTML = (" Left: " + Math.round(mxt) + "\n Top: " + Math.round(mmt) );
            }
            else {
              alert ("Your browser does not support this example!");
            }             //document.querySelector('#myraright').innerHTML = mmt;
      }
// second segment of do loop ---  start get ElementById image1
          function GetFish () {
           var div = document.getElementById ("image1");
            if (div.getBoundingClientRect) {        // Internet Explorer, Firefox 3+, Google Chrome, Opera 9.5+, Safari 4+
              var rect = div.getBoundingClientRect ();
              gx = rect.left;
              gy = rect.top;
              w = rect.right - rect.left;
              h = rect.bottom - rect.top;
              var  mt = gy+165;
              rightLip =  gx + (mt + 155) 
          //    document.getElementById("right").innerHTML = (" Right: " + gx + "\n Top: " + mt + "\n Width: " + w + "\n Height: " + h);
             document.getElementById("right").innerHTML = (" Right: " + gx + "\n Top: " + mt );
             
              if (Math.round(mxt)+1 == gx && Math.round(mmt) == mt) {
                functionADD();
                tempAlert("<h1> KISS ! </h1>",2000);
                function tempAlert(msg,duration) 
                {
                 var el = document.createElement("div");
                 el.setAttribute("style","position:absolute;top:40%;left:25%;background-image:url('heart.png');background-repeat: no-repeat;");
                 el.innerHTML = msg;
                 setTimeout(function(){
                  el.parentNode.removeChild(el);
                 },duration);
                 document.body.appendChild(el);
                }
              }
            }
            else {
              alert ("Your browser does not support this program");
            }
          }
       // End get ElementById image1 second segment of do loop ---  

       // set interval between getting position of selections 
            setInterval(GetFish, 5);
            setInterval(GetBox, 5);
            }
            while(mxt != gx && mmt != mt){
 }
 /// End of do loop

    $(document).ready(function(){
        animateDiv();
    });
function floats(){
    // Using lesson: Getting Window Dimensions
    //  var h = $(window).height() - 75;
    //  var w = $(window).width() -35; 
  // Use var h and w defined for a specific area of float
    //  var h = 100;
    //  var w = 200;
     //The +200 and +500 is to set them apart yet facing each other without too much overlap 
      var nh = Math.floor(Math.random() * h);
      var nw = Math.floor(Math.random() * w);
      return [nh,nw];    
}

function animateDivs(){
    var newq = floats();
    var oldq = $('.a').offset();
    var speed = calcSpeed([oldq.top, oldq.left], newq);
    $('.a').animate({ top: newq[0], left: newq[1] }, speed, function(){
      animateDivs();        
    });
};
//function calcSpeed(prev, next) {
  //  var x = Math.abs(prev[1] - next[1]);
  //  var y = Math.abs(prev[0] - next[0]);
  //  var greatest = x > y ? x : y;
  //  var speedModifier = .03;
  //  var speed = Math.ceil(greatest/speedModifier);
  //  return speed;
//}


    $(document).ready(function(){
    animateDivm();
 });


 $(document).ready(function(){
    animateDivC();
 });



//End Animate .c
//Start .d
 $(document).ready(function(){
    animateDivD();
 });

$(function(){
  var x = 0;
  setInterval(function(){
      x-=2;
      $('header').css('background-position', x + 'px 0');
  }, 300);
})

function App() {
  return (
    <div className="App">
      <header className="App-header">
        
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://doodad.dev/pattern-generator/"
          target="_blank"
          rel="noopener noreferrer"
        >
          Pattern Generator
        </a>

<form>
  <label>
    Name:
    <input type="text" name="name" />
  </label>
  <input type="submit" value="Submit" />
</form>

        
      </header>
    </div>
  );
}

export default App;
